concepticondata/conceptlists/Carling-2017-104a.tsv

```diff
@@ -0,0 +1,105 @@
+Carling-2017-104a-47	47	1411	LIE (REST)	to lie (recline)	1047		
+Carling-2017-104a-48	48	1224	LIVER	liver	1048		
+Carling-2017-104a-49	49	1203	LONG	long (adj)	1049		
+Carling-2017-104a-50	50	1392	LOUSE	louse (lice)	1050		
+Carling-2017-104a-51	51	1554	MAN	man (adult male human)	1051		
+Carling-2017-104a-52	52	1198	MANY	many	1052		
+Carling-2017-104a-53	53	2615	FLESH OR MEAT	meat/flesh	1053		
+Carling-2017-104a-54	54	1313	MOON	moon	1054		
+Carling-2017-104a-55	55	639	MOUNTAIN	mountain	1055		
+Carling-2017-104a-56	56	674	MOUTH	mouth	1056		
+Carling-2017-104a-57	57	1405	NAME	name	1057		
+Carling-2017-104a-58	58	1333	NECK	neck	1058		
+Carling-2017-104a-59	59	1231	NEW	new	1059		
+Carling-2017-104a-60	60	1233	NIGHT	night	1060		
+Carling-2017-104a-61	61	1221	NOSE	nose	1061		
+Carling-2017-104a-62	62	1240	NOT	not	1062		
+Carling-2017-104a-63	63	1493	ONE	one	1063		
+Carling-2017-104a-64	64	683	PERSON	person (individual human)	1064		
+Carling-2017-104a-65	65	658	RAIN (PRECIPITATION)	rain (n)	1065		
+Carling-2017-104a-66	66	156	RED	red (colour)	1066		
+Carling-2017-104a-67	67	2457	PATH OR ROAD	road/path	1067		
+Carling-2017-104a-68	68	670	ROOT	root (botanics)	1068		
+Carling-2017-104a-69	69	1395	ROUND	round	1069		
+Carling-2017-104a-70	70	671	SAND	sand	1070		
+Carling-2017-104a-71	71	1458	SAY	to say	1071		
+Carling-2017-104a-72	72	1409	SEE	to see	1072		
+Carling-2017-104a-73	73	714	SEED	seed (n)	1073		
+Carling-2017-104a-74	74	1416	SIT	to sit	1074		
+Carling-2017-104a-75	75	2613	SKIN (HUMAN)	skin (person's)	1075		
+Carling-2017-104a-76	76	1585	SLEEP	to sleep	1076		
+Carling-2017-104a-77	77	1246	SMALL	small	1077		
+Carling-2017-104a-78	78	778	SMOKE (EXHAUST)	smoke (n)	1078		
+Carling-2017-104a-79	79	1442	STAND	to stand	1079		
+Carling-2017-104a-80	80	1430	STAR	star	1080		
+Carling-2017-104a-81	81	857	STONE	stone	1081		
+Carling-2017-104a-82	82	1343	SUN	sun	1082		
+Carling-2017-104a-83	83	1439	SWIM	to swim	1083		
+Carling-2017-104a-84	84	1220	TAIL	tail	1084		
+Carling-2017-104a-85	85	78	THAT	that (deixis, distal)	1085		
+Carling-2017-104a-86	86	1214	THIS	this (deixis, proximal)	1086		
+Carling-2017-104a-87	87	1215	THOU	thou (2sg)	1087		
+Carling-2017-104a-88	88	1205	TONGUE	tongue (anatomical)	1088		
+Carling-2017-104a-89	89	1380	TOOTH	tooth (general)	1089		
+Carling-2017-104a-90	90	906	TREE	tree	1090		
+Carling-2017-104a-91	91	1498	TWO	two	1091		
+Carling-2017-104a-92	92	1443	WALK	to walk	1092		
+Carling-2017-104a-93	93	2110	WARM (OF WEATHER)	warm (of weather)	1093		
+Carling-2017-104a-94	94	948	WATER	water (n)	1094		
+Carling-2017-104a-95	95	1212	WE	we (1pl)	1095		
+Carling-2017-104a-96	96	1236	WHAT	what?	1096		
+Carling-2017-104a-97	97	1335	WHITE	white (colour)	1097		
+Carling-2017-104a-98	98	1235	WHO	who?	1098		
+Carling-2017-104a-99	99	962	WOMAN	woman	1099		
+Carling-2017-104a-100	100	1424	YELLOW	yellow (colour)	1100		
+Carling-2017-104a-101	101	2099	CHILD	child	1101		
+Carling-2017-104a-102	102	1131	WE (INCLUSIVE)	we (1pl incl.)	1103		
```

Comment:
Ah, they deliberately extend swadesh by exclusive, inclusive, etc? So here you have what to write in the little description.